### ISSUE TITLE
All LR Commands: Move [Credential] parameter to the end of each parameter set.

### ISSUE THREAD

**Issue by GeneCupstid** [LogRhythm-Tools/LogRhythm.Tools#19]
Currently it is first, but will rarely be used since the configuration handles API keys.  For convenience move this parameter to the end of each cmdlet.

**Comment by GeneCupstid** [comment]
This also causes issues like the following:

```
> Get-LrCaseById 3754
Get-LrCaseById : Cannot process argument transformation on parameter 'Credential'. userName
At line:1 char:16
+ Get-LrCaseById 3754
+                ~~~~
    + CategoryInfo          : InvalidData: (:) [Get-LrCaseById], ParameterBindingArgumentTransformationException
    + FullyQualifiedErrorId : ParameterArgumentTransformationError,Get-LrCaseById
```

**Comment by Jt3kt** [comment]
Yes please.  I'd shoot for 1.0 as possible.

**Comment by GeneCupstid** [comment]
It will def. be 1.0, and I will implement this as soon as PR #17 is completed!  :neckbeard:

**Comment by GeneCupstid** [comment]
This was resolved with credential update PR #32
